app/assets/stylesheets/print.css.scss

```diff
@@ -0,0 +1,139 @@
+/* -*- coding: utf-8 -*- */
+
+/**
+ * Angaben zum Ausdruck von Postings
+ * 
+ * @author: Heiko August <post@auge8472.de>
+ * @author: Matthias Apsel
+ * @version: 0.2
+**/
+
+.printfootnoteurls {
+  margin: 1em 0;
+  padding-left: 0;
+  
+  li {
+    list-style: none;
+  }
+
+  sup.printfootnote {
+    top: 0;
+  }
+}
+
+.print-preview .thread-message.print .posting-content a {
+  color: inherit;
+}
+
+@media print {
+  body.print-preview {
+    font-size: 1rem;
```

Comment:
Nein.

~~~
.class {
  eigenschaft1: wert;

  element {
    eigenschaft2: wert;
  }
}
~~~

ist gültiges SASS.

resultierend in

~~~
.class { eigenschaft1: wert; }
.class element { eigenschaft2: wert; }
~~~